packages/keystone/src/lib/core/types-for-lists.ts

```diff
@@ -262,7 +312,34 @@ export function initialiseLists(
             if (typeof fieldFunc !== 'function') {
               throw new Error(`The field at ${listKey}.${fieldKey} does not provide a function`);
             }
-            return [fieldKey, fieldFunc({ fieldKey, listKey, lists: listInfos, provider })];
+            let f = fieldFunc({ fieldKey, listKey, lists: listInfos, provider });
+
+            const isEnabled = {
+              read: f.graphql?.isEnabled?.read ?? true,
+              update: f.graphql?.isEnabled?.update ?? true,
+              create: f.graphql?.isEnabled?.create ?? true,
+              filter: f.graphql?.isEnabled?.filter ?? false,
+              orderBy: f.graphql?.isEnabled?.orderBy ?? false,
+            };
+            const field = { ...f, graphql: { ...f.graphql, isEnabled } };
```

Comment:
(I forgot this in the diff I sent before I edited it)
```suggestion
            const field = { ...f, graphql: { ...f.graphql, isEnabled }, input: { ...f.input } };
```